app/services/importers/doc/alias.rb

```diff
@@ -4,7 +4,7 @@ class Alias
       attr_accessor :file, :fields, :field_pattern, :bar, :doc_mapping, :dir
 
       def initialize(dir)
-        @file = Bucket.new.get_object("doc/#{dir}/Vendor_Alias_Extract_Text.dat")
+        @file = Bucket.new.get_object("doc/#{dir}/vendor_alias_extract_text.dat")
```

Comment:
converting these to lowercase in quarterly_data.rb to deal with inconsistent casing (they're frequently suffixed with .DAT)